docker-compose.yml

```diff
@@ -1,48 +1,55 @@
 version: "3.7"
-
+networks:
```

Comment:
Maybe it's just a matter of taste, but please bring back empty lines dividing service definitions for the sake of readability/manageability. E.g.:
```yml
version:

networks:
# networks definition

service-a:
#definition 

service-b:
#defintion

volumes:
#definition
```

Reply:
we have 10+ repositories with docker-compose files. Let's format zebrunner into the common org format:)